media-libs/libgroove/libgroove-4.3.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="Streaming audio processing library."
+HOMEPAGE="https://github.com/andrewrk/libgroove"
+SRC_URI="https://github.com/andrewrk/libgroove/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
```

Comment:
I see that it installs both `libgroove.so.4` and `libgroove.so.4.3.0`. How do we decide between `0/4` and `0/4.3.0`?